packages/lesswrong/server/sql/SqlFragment.ts

```diff
@@ -43,6 +43,9 @@ const getResolverCollection = (
   if (exclam >= 0) {
     type = type.substring(0, exclam);
   }
+  while (type[0] === "[") {
```

Comment:
This PR is the first time we've added a SQL resolver that returns an array of objects, which it turns out required a couple of changes here.